src/H5FDlog.c

```diff
@@ -502,6 +502,10 @@ H5FD__log_open(const char *name, unsigned flags, hid_t fapl_id, haddr_t maxaddr)
     if (ADDR_OVERFLOW(maxaddr))
         HGOTO_ERROR(H5E_ARGS, H5E_OVERFLOW, NULL, "bogus maxaddr")
 
+    /* Initialize timers */
+    H5_timer_init(&open_timer);
+    H5_timer_init(&stat_timer);
+
```

Comment:
Incidentally, this is one of those places where it would make sense to intermingle declarations and statements as we could declare the timer and then immediately initialize it.